nfp/preprocessing/xtb_preprocessor.py

```diff
@@ -49,9 +49,23 @@ def __init__(
                 "Dispersion coefficient C6",
                 "Polarizability alpha",
             ]
+<<<<<<< HEAD
```

Comment:
this looks like the result of a not-quite complete git merge ...

Reply:
You mean with xtb? I don't do it on Jupyter Notebooks though